system/screen.h

```diff
@@ -46,9 +46,21 @@ typedef enum {
 } screen_colour_t;
 
 /**
- * BIOS/UEFI(GOP) agnostic framebuffer copy
+ * Colour Palette definition
  */
+typedef struct {
+    screen_colour_t background;
+    screen_colour_t foreground;
+    screen_colour_t title_background;
+    screen_colour_t title_foreground;
+    screen_colour_t footer_background;
+    screen_colour_t footer_foreground;
+    screen_colour_t popup_background;
```

Comment:
Do we also want another set for a settings/menu? In the "classic" one the text is white, disabled is gray, error text is yellow, bg is black. If we already have black background, we probably want this to be inverted? Background - white, text  -black, not sure about f_menu_error, f_menu_disabled?

Reply:
Tested menus with inverted colors, it doesn't look good. Black background for popups looks fine, even on dark mode.